### DISCUSSION TITLE
Suggestion to provide explicit getter for `registry.proxies`

### DISCUSSION THREAD

**Discussion by razgraf** [PaulRBerg/prb-proxy#105] [Ideas]
The v4.0.0-beta.5 update replaced the [proxy mapping accessor](https://github.com/PaulRBerg/prb-proxy/blob/6fbb6df62d25e07fc702a4485595aa34d8297237/src/PRBProxyRegistry.sol#L43-L45). What used to be a dedicated `getProxy` method has now been replaced by an auto-generated getter called `proxies`.

I'd suggest reverting back to an explicit `getProxy` (or `proxyOf`) method. 

While the name of the variable is still contextually correct, when used as a getter it implies plurality. When queried/read, the contract interface should make it clear that a user holds a single `proxy` not multiple `proxies`.

<details>
<summary>Example of the API as shown on Etherscan</summary>

![Etherscan](https://github.com/PaulRBerg/prb-proxy/assets/15740806/8b8b0193-b7c1-4f75-bd85-878141f2fa72)
</details>

**Comment by PaulRBerg** [comment]
Yep; fully agree that `getProxy` is clearer, and that the extra getter is worth it.

Let's wait for the audit to finish, and I will try to include this in the post-audit changes.

**Comment by PaulRBerg** [comment]
Implemented via https://github.com/PaulRBerg/prb-proxy/pull/118.
